Demo/UI Tests/ThreeDSecure UI Tests/ThreeDSecure_V2_UITests.swift

```diff
@@ -13,19 +13,21 @@ class ThreeDSecure_V2_UITests: XCTestCase {
         app.launchArguments.append("-Integration:BraintreeDemoThreeDSecurePaymentFlowViewController")
         app.launch()
 
-        _ = app.cardNumberTextField.waitForExistence(timeout: 10)
+        waitForElementToAppear(app.cardNumberTextField)
```

Comment:
The 3DS test need to be run with a client token (flow does not work with tokenization key). The tests were timing out due to not waiting long enough for the button to appear. Using `waitForElementToAppear ` allows us to wait for the client token to be generated from the server and the buttons to appear. A sleep has been added to tests where not present for consistency.

Reply:
We have some hard coded client tokens for our integration tests ([see code](https://github.com/braintree/braintree_ios/blob/803a41d3ad64b0792e755c37400947154911eefb/IntegrationTests/BraintreePayPal_IntegrationTests.swift#L16-L39)). We can see if these are usable in the UI tests as well.

Reply:
That's a good call - added a launch argument to use a hardcoded client token in UI tests here: https://github.com/braintree/braintree_ios/pull/1067/commits/1e482626fbedd946ad7fe069918d337842d43995